.github/workflows/preview-close.yml

```diff
@@ -0,0 +1,37 @@
+name: 'Amplify PR Closed'
+on:
+  pull_request_target:
+    branches:
+      - main
```

Comment:
Is this filter needed? I assume that if we had a PR from `other-branch` into `feature-branch`, we would still want to delete the preview environment when `other-branch` merges.